tests/utils/is-empty.test.ts

```diff
@@ -31,7 +31,7 @@ describe('isEmpty', () => {
       expect(t.isEmpty(() => {})).to.strictlyEqual(true);
 
       // deleting all the keys from on object empties it
-      o = { a: true };
+      o = { a: true } as { a?: boolean };
```

Comment:
Same note about `as` casting.
```suggestion
      const o: { a?: boolean } = { a: true };
```